### PR TITLE
 3618 fix visual bug in CattegoryListView

### PR DESCRIPTION
**Pull request by AmirShelli**
Remove margin in cattributesBox, see #3618

### CODE REVIEW

**Comment by KremnevDmitry** [comment]
Please note that the original issue listed three problems
This pull request only addresses one of them

File: jmix-dynattr/dynattr-flowui/src/main/resources/io/jmix/dynattrflowui/view/category/category-list-view.xml [hunk 1]
```diff
@@ -75,7 +75,7 @@
                     </columns>
                 </dataGrid>
             </vbox>
-            <vbox id="attributesBox" margin="true" padding="false" height="100%" expand="attributesTable">
+            <vbox id="attributesBox" margin="false" padding="false" height="100%" expand="attributesTable">
```
Comment (KremnevDmitry) [inline jmix-dynattr/dynattr-flowui/src/main/resources/io/jmix/dynattrflowui/view/category/category-list-view.xml:78]: The `margin` attribute for `ThemableLayout` has a `false` default value. It doesn't need to be specified explicitly, it is enough to remove the attribute

**Review by KremnevDmitry — CHANGES_REQUESTED** [review]

**Comment by AmirShelli** [comment]
Removing the margin attribute got rid of 2 issues: 
1. extension beyond view boundaries
2. redundant padding
Adding `themeNames="splitter-spacing"` created space between the split
Screenshot 2024-09-13 at 14 43 09

File: jmix-dynattr/dynattr-flowui/src/main/resources/io/jmix/dynattrflowui/view/category/category-list-view.xml [hunk 2]
```diff
@@ -40,8 +40,8 @@
         <dataLoadCoordinator auto="true"/>
     </facets>
     <layout spacing="true">
-        <split id="split" orientation="HORIZONTAL" width="100%" minHeight="25em" height="100%">
-            <vbox id="categoryBox" margin="false" padding="false" height="100%" expand="categoriesGrid">
+        <split id="split" orientation="HORIZONTAL" width="100%" minHeight="25em" height="100%" themeNames="splitter-spacing">
```
Comment (KremnevDmitry) [inline jmix-dynattr/dynattr-flowui/src/main/resources/io/jmix/dynattrflowui/view/category/category-list-view.xml:43]: This line is beyond the 120 character limit
Please break the line

image

**Pull request merged**